src/std/net/smtp/README.org

```diff
@@ -0,0 +1,902 @@
+#+TITLE: SMTP: Simple Mail Tranfser Protocol
+
```

Comment:
Bigger issue with org-mode: maintenance.

Unless the system enforces generation from the org file that becomes the source code (at which point the generated files are excluded from git), then what of this redundant representation of code? Are maintainers going to keep the two in sync? Edit only the org-mode then regenerate the code? (how?) Edit only the code and let the org-mode stay out of date?

Reply:
The answer "for now, drewc is maintainer" is fine with me if it's fine with @vyzo — but then it needs a comment on the top of each and every relevant `.ss` file.